Homework_8/functional.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[212]:
+
+
+def sequential_map(*args, str_iter=True):
+    container = args[-1] 
```

Comment:
Вот так удобнее, потом не придётся брать срезы от args
```suggestion
    *functions, container = args
```